flatdata-rs/lib/src/storage.rs

```diff
@@ -195,7 +193,7 @@ pub fn check_resource<T>(
 /// storage.
 #[doc(hidden)]
 pub fn create_external_vector<'a, T>(
-    storage: &'a dyn ResourceStorage,
+    storage: &'a (dyn ResourceStorage + std::marker::Sync + std::marker::Send),
```

Comment:
I think Sync and Send are in prelude, so technically no fqn is needed.

Reply:
Done